recirq/qcqmc/hamiltonian.py

```diff
@@ -0,0 +1,319 @@
+from pathlib import Path
+from typing import Tuple, Union
+
+import attrs
+import fqe
+import h5py
+import numpy as np
+import openfermion as of
+from fqe.hamiltonians.restricted_hamiltonian import RestrictedHamiltonian
+from fqe.openfermion_utils import integrals_to_fqe_restricted
+from openfermionpyscf._run_pyscf import compute_integrals  # type: ignore
```

Comment:
Why are we importing from a private module?  This is generally not advisable.

Reply:
I copied the openfermion function into hamiltonian.py, WDYT?